spec/spec_helper.rb

```diff
@@ -0,0 +1,100 @@
+# This file was generated by the `rspec --init` command. Conventionally, all
```

Comment:
It would be good to see your coverage on your rspec tests. You can use the simplecov console gem and add some lines in here to see that.